src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
+        Machine[] machines = {bulldozer, excavator, truck};
+        for (Machine machine: machines) {
+            machine.doWork();
```

Comment:
The `doWork` method should output an informative message indicating what type of `Machine` is working (e.g., `Truck`, `Bulldozer`, or `Excavator`). Make sure that the `doWork` method in each specific machine class outputs a message that includes the machine type.